doc/version_history.rst

```diff
@@ -2,6 +2,20 @@
 Version History
 ===============
 
+v0.4.1
+------
+* In CSC:
+  * Refactor how telemetry/event loops start/stop.
+  * Refactor handle_summary_state.
+  * Use begin_<cmd> to report state transition ack in progress for all state commands.
+* In tcp_client, don't write eof when closing connection and log when connection is closed.
```

Comment:
As I wrote in the another comment, if you do not write the EOF from client, the server side in LabVIEW will not know the connection has been closed by the client.